Basic/class/class.py

```diff
@@ -7,12 +7,12 @@ class Hewan(ABC):
     # Properti absstrak
     @abstractproperty
     def suara(self):
-        ...
```

Comment:
Tbh it's fine as-is.

Reply:
iya di lint pyflake8 juga bisa tapi di CI nya error, jadi kuganti pass

Reply:
Weird. `pass` itu sendiri juga gada efek.